src/db/sysdb_search.c

```diff
@@ -2494,6 +2495,11 @@ errno_t sysdb_get_direct_parents(TALLOC_CTX *mem_ctx,
     DEBUG(SSSDBG_TRACE_INTERNAL,
           "searching sysdb with filter [%s]\n", member_filter);
 
+    if (attr_name == NULL) {
+        attr_name = SYSDB_NAME;
+    }
```

Comment:
what about passing `SYSDB_NAME` as an argument in `sysdb_get_direct_parents()`  instead, and fail if attr_name is not provided in `sysdb_get_direct_parents_ex()` ? I don't insist however.

Reply:
Hi,

thanks for the suggestion, I was thinking about this as well but decided against because using SYSDB_NAME is imo a sensible default and by using this default if no attribute name is giving we can avoid failing with an error.

bye,
Sumit